src/components/core/Tab.js

```diff
@@ -1,15 +1,29 @@
-import React from 'react';
-import { withStyles } from '@material-ui/core/styles';
-import { Tab as MuiTab } from '@material-ui/core';
-
-const styles = {
-    root: {
-        minWidth: 0,
-        fontSize: 14,
-    },
+import React, { useContext, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { Tab as UiTab } from '@dhis2/ui';
+import { TabContext } from './Tabs';
+
+const Tab = ({ value, dataTest, children }) => {
+    const { tab, onChange } = useContext(TabContext);
+
+    // onChange is from the parent component
+    const onClick = useCallback(() => {
```

Comment:
I'm not very familiar with `useCallback` yet.
Is this really required here?

Reply:
useCallback removed